### ISSUE TITLE
train_bottleneck both loss and val_loss are always nan

### ISSUE THREAD

**Issue by monicatao** [qqwweee/keras-yolo3#715]
Hi everyone, I am training the model on my own dataset and I only have one class. anchors file is not changed. And I tried to train with and without pretrained.h5. But my loss and val_loss are always nan. Anyone have similar problem and the solution? Thank you!
![Screenshot 2020-08-10 at 21 05 31](https://user-images.githubusercontent.com/22206438/89820705-b7276b00-db4d-11ea-9551-41bc68a865af.png)

**Comment by ouarda253** [comment]
@monicatao  i have the same probleme did you solve it ?

**Comment by heshuju** [comment]
I only have val_loss is nan, Because initial weights is not fit, and I use weights from Official website to solve it
